governance/polls/Ratification Poll - Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Ratification Poll for Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021
+summary: Signal your support or opposition for this proposal being accepted and implemented in the Maker Protocol. 
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp17-sustainable-ecosystem-scaling-core-unit-mkr-budget-ses-001/8043
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Ratification Poll for Sustainable Ecosystem Scaling Core Unit MKR Budget, SES-001 (MIP40c3-SP17) - June 14, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to this proposal being accepted and implemented in the Maker Protocol.
+
+The proposal in its entirety [can be found here](https://github.com/makerdao/mips/blob/40de9327bb3df265f6074c32710b68f2b5bb681e/MIP40/MIP40c3-Subproposals/MIP40c3-SP17.md). The sentence summary and paragraph summary have been provided by the MIP author:
+
+**Sentence Summary**
+
+*MIP40c3-SP13 adds the budget for Core Unit RISK-001: Risk.*
+
+**Paragraph Summary**
+
+_MIP40c3-SP17 adds the MKR Incentive Plan budget for Core Unit SES-001: Sustainable Ecosystem Scaling. It contains:_
+
+- _Total MKR Expenditure Cap_
+- _Estimated MKR Expenditure (based on the current team)_
+- _Escrow Wallet mechanism_
+
+_MKR incentives have been determined based on the [Program discussed here](https://forum.makerdao.com/t/pre-mip-discussion-an-alternative-mkr-compensation-plan/8000). This is a 3-year vesting plan with 1-year cliff vest._
+
+Please review the discussion [thread](https://forum.makerdao.com/t/mip40c3-sp13-modify-core-unit-budget-risk-001/8034) to inform your position on this proposal before voting. It was also mentioned and possibly discussed on the [Governance and Risk call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-145-thursday-june-10-17-00-utc/8590) on Thursday, June 10.
```

Comment:
Wrong thread link. Should be: https://forum.makerdao.com/t/mip40c3-sp17-sustainable-ecosystem-scaling-core-unit-mkr-budget-ses-001/8043